tensorrt_llm/_torch/auto_deploy/distributed/__init__.py

```
@@ -0,0 +1,4 @@
+"""Distributed operations and backend management."""
+
+# Import trtllm to ensure custom ops are registered
+from . import trtllm  # noqa: F401
```

Ruff (0.14.4)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)